### DISCUSSION TITLE
Welcome to udm-utilities Discussions!

### DISCUSSION THREAD

**Discussion by boostchicken** [unifi-utilities/unifi-common#108] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by qwerk** [comment]
HI, udm-utils worked very nice on udm 1.8.6  just updated to 1.9.0.10  now my iptables does not get updated anymore, as far as I can see.
Is IPtables changed in the new firmware? Do I need to change something?

**Comment by alxwolf** [comment]
Hi @boostchicken, thank you for providing us the means to further improve our UDM.

I'm with UniFi gear since years, and annoyed about them ignoring TLS security. 
This led me to sharing an approach to Let's Encrypt Certificates using acme.sh instead of LEGO (which is used by @kchristensen's udm-le, but cannot handle my provider's DNS API).

Everyone's welcome to have a look at [ubios-cert](https://github.com/alxwolf/ubios-cert) and improve it, if desired. Once I've received at least some feedback, I will consider a PR for a link in your repo.

I just spent several hours trying to get myself acquainted with Github, VS Code and shell scripts. My most recent toolchain was based on Emacs and RCS for C++ and ... Fortran. So, please be nice and patient with me :)
